src/containers/MathObjects/components/StatusSymbol/components/ColorSquare.js

```diff
@@ -19,34 +20,16 @@ export default class ColorSquare extends React.PureComponent {
     onPickColor: PropType.func.isRequired
   }
 
-  state = {
-    hover: false
-  }
-
-  startHover = () => {
-    this.setState( { hover: true } )
-  }
-
-  stopHover = () => {
-    this.setState( { hover: false } )
-  }
-
-  onPointerUp = () => {
+  pickColor = () => {
     this.props.onPickColor(this.props.color)
   }
 
   render() {
     return (
-      <Pointable
-        onPointerUp={this.onPointerUp}
-        onPointerOver={this.startHover}
-        onPointerOut={this.stopHover}
-      >
-        <Color
-          color={this.props.color}
-          hover={this.state.hover}
-        />
-      </Pointable>
+      <Color
+        color={this.props.color}
+        onClick={this.pickColor}
```

Comment:
@ChristopherChudzicki How would I do the color picking with the styled component alone? onPickColor needs to be called with the `this.prop.color` as argument. So I think I need the extra component for that.